### PR TITLE
Make operator watch in all namespaces.

### LINKED ISSUE #8: Intermittent issue - Status attribute is not always updated when secret is created.
Seeing an intermittent issue where the credential attribute is not populated when a secret is created:

```
spec:
  dbName: postgres
  image: docker.io/postgres
  imageName: postgres
status:
  dbConnectionIP: 172.30.105.222
  dbConnectionPort: 5432
  dbCredentials: ""
  dbName: postgres
```
The dbCredentials points to a secret that the db operator creates, so the reason why it is empty is probably the failed update of the status field after the secret was created. (Tt happens sometimes and most of the time deletion and re-creation of the DB CR helps).

### PR DESCRIPTION
**Pull request by pmacik**
This PR:
* Introduces optional `OPERATOR_TESTING` variable for the `Makefile` to build/package/push to alternative `*-testing` repo
* Enables the operator to watch all namespaces
* Tweaks the PostgreSQL container spec to work with other namespaces
* Attempts to fix https://github.com/baijum/postgresql-operator/issues/8

### CODE REVIEW

File: manifests/database.package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-packageName: db-operators
+packageName: REPLACE_PACKAGE
 channels:
-- currentCSV: postgresql-operator.v0.0.3
+- currentCSV: REPLACE_NAME.vREPLACE_VERSION
```
Comment (otaviof) [inline manifests/database.package.yaml:3]: Would be nice to not have this `vREPLACE_VERSION` to only keep `REPLACE_VERSION`. Another approach would be adding `REPLACE_IMAGE_TAG` where you bring together the prefix you would need to reflect the image tag.

File: Makefile [hunk 2]
```diff
@@ -1,11 +1,16 @@
 
 QUAY_USERNAME ?= 
 QUAY_PASSWORD ?= 
-POSTGRESQL_OPERATOR_VERSION ?= 0.0.3
-POSTGRESQL_OPERATOR_IMAGE ?= quay.io/$(QUAY_USERNAME)/postgresql-operator
+
+ifdef OPERATOR_TESTING
+POSTGRESQL_APPR_NAMESPACE ?= $(QUAY_USERNAME)-testing
```
Comment (otaviof) [inline Makefile]: As a general suggestion, I think the word `POSTGRESQL_` in front of every variable is not needed, you could very well remove it.

File: deploy/crds/postgresql_v1alpha1_database_cr.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: postgresql.baiju.dev/v1alpha1
 kind: Database
 metadata:
   name: demo-database
-  namespace: openshift-operators
+  namespace: demo-app
```
Comment (otaviof) [inline deploy/crds/postgresql_v1alpha1_database_cr.yaml:5]: What's the meaning of `demo-app` hardcoded as namespace? Would it apply to have a `REPLACE_NAMESPACE` following the other variable names?

File: Makefile (hunk 2 above)
Comment (pmacik) [reply Makefile]: Good point... removed
Thanks.

File: deploy/crds/postgresql_v1alpha1_database_cr.yaml (hunk 3 above)
Comment (pmacik) [reply deploy/crds/postgresql_v1alpha1_database_cr.yaml:5]: This `deploy/crds/postgresql_v1alpha1_database_cr.yaml` is just an example CR, isn't it?
The operator can create CRs in any namespace - so `demo-app` is an example.

On the other hand, the namespace used at `REPLACE_NAMESPACE` by the `Makefile` is the `Quay.io`'s application namespace - basically quay username or org.

File: manifests/database.package.yaml (hunk 1 above)
Comment (pmacik) [reply manifests/database.package.yaml:3]: @otaviof Interesting idea, but let's solve this via another PR, the current solution, while not being ideal, still works.

**Review by baijum — APPROVED** [review]

**Pull request merged**